rs/embedders/Contributing.md

```diff
@@ -0,0 +1,11 @@
+= Introducing a new System API or changing an existing one:
+
+1. Specify the new API or make the required changes to an existing one in the [portal](https://github.com/dfinity/portal) repository. Typically, you would want to make changes in the [following file](https://github.com/dfinity/portal/blob/master/docs/references/ic-interface-spec.md). Make sure to provide motivation for the new API or changes you're making and how it would benefit the ICP protocol.
+2. The Rust CDK would need to be updated: [Rust Canister Development Kit](https://github.com/dfinity/cdk-rs). Inform *@eng-sdk* of the work required.
+3. Motoko would need to be updated: [Motoko](https://github.com/dfinity/motoko). Inform *@eng-motoko* of the work required.
+4. Implement the new API in `embedders` crate. Main parts that need to be updated: the wasm validation, the linker of the system API against the Wasm module and the implementation of the system api itself.
```

Comment:
what do you mean by "wasm validation" here? isn't validation implicit by adjusting the linker?